### PR TITLE
[SLATE]: Journeys example implemented (jsFiddle)

### PR DESCRIPTION
**Pull request by lrocheWB**
@stifoon :)

### CODE REVIEW

File: documentation/slate/source/examples/jsFiddle/journeys/demo.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ * is only an example for demo purposes an does not reflect
+ * the actual transportation supply on the Parisian area.
+ */
+
+$.ajax({
+  type: 'GET',
+  url: 'https://api.navitia.io/v1/journeys?from=2.3749036;48.8467927&to=2.2922926;48.8583736',
+  dataType: 'json',
+  headers: {
+    'Authorization': 'Basic ' + btoa('a61f4cd8-3007-4557-9f4c-214969369cca')
```
Comment (antoine-de) [inline documentation/slate/source/examples/jsFiddle/journeys/demo.js]: what token is this ?

File: documentation/slate/source/examples/jsFiddle/journeys/demo.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+ * It requests an itinerary from a cool spot (our headquarter in Paris)
+ * to another cool spot (the Eiffel Tower, Paris).
+ * Please note that the transportation data that is used for this sandbox
+ * is only an example for demo purposes an does not reflect
+ * the actual transportation supply on the Parisian area.
+ */
+
+$.ajax({
+  type: 'GET',
+  url: 'https://api.navitia.io/v1/journeys?from=2.3749036;48.8467927&to=2.2922926;48.8583736',
```
Comment (antoine-de) [inline documentation/slate/source/examples/jsFiddle/journeys/demo.js]: don't we want to enforce that the journey is done on the sandbox coverage ?

**Comment by antoine-de** [comment]
@lrocheWB can you render the doc on your repo so that we can have a look at this ?

seems great!

File: documentation/slate/source/examples/jsFiddle/journeys/demo.details [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+name: Journeys API - jsFiddle/Github
+description: |
+    The following example aims at illustrating Navitia’s journey feature.
+    It requests an itinerary from a cool spot (our headquarter in Paris) to another cool spot (the Eiffel Tower, Paris).
+    Please note that the transportation data that is used for this sandbox is only an example for demo purposes an does not reflect the actual transportation supply on the Parisian area.
+authors:
+   - Julien Maulny
+   - Ludovic Roche
+normalize_css: no
+...
```
Comment (pbougue) [inline documentation/slate/source/examples/jsFiddle/journeys/demo.details]: missing newline?

File: documentation/slate/source/examples/jsFiddle/journeys/demo.js (hunk 1 above)
Comment (pbougue) [inline documentation/slate/source/examples/jsFiddle/journeys/demo.js]: Yep, maybe you can use the dummy token `123456789abcdef-blablabla` that is already mentionned in doc

File: documentation/slate/source/stylesheets/navio.css.scss [hunk 4]
```diff
@@ -23,3 +23,28 @@
...
+    text-transform: uppercase;
+    transition: background-color .2s,box-shadow .2s;
+    vertical-align: middle;
+    white-space: nowrap;
+    box-shadow: 0 2px 5px 0 rgba(0,0,0,.26);
+}
+
+.button-blue {
+    color: #039be5;
+}
```
Comment (pbougue) [inline documentation/slate/source/stylesheets/navio.css.scss]: no newline

**Comment by pbougue** [comment]
Would be nice to have a glance at it first, but seems ok for me

File: documentation/slate/source/stylesheets/navio.css.scss (hunk 4 above)
Comment (lrocheWB) [inline documentation/slate/source/stylesheets/navio.css.scss]: done :)

File: documentation/slate/source/examples/jsFiddle/journeys/demo.details (hunk 3 above)
Comment (lrocheWB) [inline documentation/slate/source/examples/jsFiddle/journeys/demo.details]: done :)

File: documentation/slate/source/examples/jsFiddle/journeys/demo.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
+body {
+  font-family: Helvetica, Sans, Arial;
+}
+
+p:first-child {
+  margin: 20px;
+}
+
+table {
+    font-size: 16px;
```
Comment (TeXitoi) [inline documentation/slate/source/examples/jsFiddle/journeys/demo.css]: CSS in px ? Em and ex are better units especially these days with retina qhd too much pixels...

**Comment by lrocheWB** [comment]
@antoine-de could you remove 'do_not_merge' label ? Ready for review and merge :)

File: documentation/slate/source/examples/jsFiddle/journeys/demo.css (hunk 5 above)
Comment (lrocheWB) [inline documentation/slate/source/examples/jsFiddle/journeys/demo.css]: Done

**Comment by alcalyn** [comment]
You can preview JsFiddle exemple here on fork: http://jsfiddle.net/gh/get/jquery/2.2.2/lrocheWB/navitia/tree/documentation/slate/source/examples/jsFiddle/journeys/

File: documentation/slate/source/examples/jsFiddle/journeys/demo.details [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (pbougue) [inline documentation/slate/source/examples/jsFiddle/journeys/demo.details:1]: maybe add licence? I don't know if it's necessary here...

**Comment by pbougue** [comment]
Ok for me, @colinele I let you have a look at the example and tell if we want a licence in that code too...

**Comment by alcalyn** [comment]
@colinele 

License: https://github.com/lrocheWB/navitia/blob/dev/documentation/slate/source/examples/jsFiddle/journeys/demo.details

Exemple: http://jsfiddle.net/gh/get/jquery/2.2.2/lrocheWB/navitia/tree/documentation/slate/source/examples/jsFiddle/journeys/

File: documentation/slate/source/examples/jsFiddle/journeys/demo.details (hunk 6 above)
Comment (lrocheWB) [inline documentation/slate/source/examples/jsFiddle/journeys/demo.details:1]: done

**Comment by pbougue** [comment]
Jenkins, test this please.

**Pull request merged**